src/static/js/modules/secondary-nav-toggle.js

```diff
@@ -25,11 +26,10 @@ function _navSecondaryToggle() {
 
 // Tests whether or not the secondary nav should be toggled.
 function _navSecondaryToggleTest() {
-  var isSmall =
-    $( '.nav-secondary .nav-secondary_link__button' ).is( ':visible' );
+  var isSmall = getBreakpointState().isSmall;
```

Comment:
An example of my question below is that you don't know the value of isSmall without looking at `get-breakpoint-state.js` (as a new dev, I'd assume it was just the mobile size in the config). Defining it here as `responsiveState.isMobile || responsiveState.isTablet` removes any questions.